Resources/views/seometas_metaslinks.html.twig

```diff
@@ -46,12 +46,19 @@
 
 {% if  ezpublish.availableLanguages|length > 1 %}
     {% for lang in ezpublish.availableLanguages %}
-        {% if location is defined and not location.isDraft %}
+        {% set href_lang = ezpublish.configResolver.parameter('href_lang', 'nova_ezseo', ezpublish.translationSiteAccess( lang )) %}
+        {% if href_lang is not empty %}
+            <link rel="alternate" hreflang="{{ href_lang['href'] }}" href="{{ url( location, {"siteaccess": ezpublish.translationSiteAccess( lang ) } ) }}" />
```

Comment:
if it's a draft I don't now if {{ url( location, {"siteaccess": ezpublish.translationSiteAccess( lang ) } ) }}
return an error ?
So maybe add on your condition if( href_lang is not empty)
 and not location.isDraft, to redirect on the last else.